### PR TITLE
Typos from marvin.cs.uidaho.edu: B

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR #1951. I took care of the comments from the code review, and also added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4688,7 +4722,9 @@ behaviuor->behaviour
 behavoir->behavior
 behavoirs->behaviors
 behavor->behavior, behaviour,
+behavoral->behavioral
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:4725]: `behavoral->behavioral, behavioural,` for consistency with the previous line?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5243,12 +5323,15 @@ brockened->broken
 brocolee->broccoli
 brocoli->broccoli
 brodcast->broadcast
+broge->brogue
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5326]: I have just learned `brogues` can be shoes, formerly worn in parts of Ireland and the Scottish Highlands.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:4725]: Makes sense, yes. Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:5326]: Thanks, now I've also learned that. I'm sure someone out there will make good use of this correction. ;-)

I've added the plural form `broges->brogues` too.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**